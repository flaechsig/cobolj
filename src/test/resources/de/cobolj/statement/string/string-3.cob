IDENTIFICATION DIVISION. 
    PROGRAM-ID. HELLO-WORLD.
    
DATA DIVISION.
WORKING-STORAGE SECTION.
  01 WS-RESULT PIC A(11).
  
PROCEDURE DIVISION.
	STRING 'HELLO' ' ' 'WORLD' DELIMITED SIZE 
	    INTO WS-RESULT
	    ON OVERFLOW 
	        DISPLAY 'OVERFLOW ERROR'
	        DISPLAY 'SECOND LINE'
	    NOT ON OVERFLOW 
	        DISPLAY 'NO OVERFLOW ERROR'
	        DISPLAY 'SECOND LINE (NO ERROR)'
	END-STRING.
    DISPLAY WS-RESULT.