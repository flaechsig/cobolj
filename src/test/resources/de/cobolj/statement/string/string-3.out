NO OVERFLOW ERROR
SECOND LINE (NO ERROR)
HELLO WORLD
